tests/runtime/multichain.jdf

```diff
@@ -233,7 +249,8 @@ int main(int argc, char* argv[])
             PARSEC_CHECK_ERROR(rc, "parsec_context_wait");
             TIMER_STOP(time_elapsed);
 
-            printf("Comm %s (test %d) Loop %d DAG execution in %ld micro-sec\n", comm_name[i], i, l, time_elapsed);
+            printf("Comm %s (test %d) Loop %d DAG execution in %ld micro-sec [world %d/%d]\n",
```

Comment:
Do we want to keep the printfs here?

Reply:
it's in the 'app' so it's not abnormal that we use printfs.